extension/content_script.js

```diff
@@ -635,21 +640,64 @@ function toggleExtensionVisibility() {
 
 function setupListenerForOnOff() {
 	chrome.runtime.onMessage.addListener(	
-		function(request, sender, sendResponse) {	
-			if (request.command === "toggleUI") {		
+		function(request, sender, sendResponse) {
+			function toggleApp() {
 				if (doc === null) {
 					// Make sure these load after animation
-					$("#loadingIcon").show(100, function() {
+					$("#loadingIcon").show(500, function() {
 						preprocessPage();
 						toggleExtensionVisibility();
 					});	
 				}
 				else { toggleExtensionVisibility(); }
+			}			
+			if (request.command === "toggleUI") {
+				toggleApp();
+			}	
+			// TODO: This can be perhaps refactored. For now, it's an easy fir
+			else if (request.command === "toggleUITutorial") {
+				if (settings.getAppStatus()) {
+					return; // If app status is on, do nothing
+				} 
+				toggleApp();
 			}	
+			
 		}	
 	);
 };
 
+function setupTutorial() {
+	// TODO: Move these into separate file
+	let tutorialPopupHtml = `
+		<div id="tutorialPopupContainer">
+			<div class="popupText">Click the<span id="optionsButtonTutorial">${window.gearLogo}</span>for instructions</div>
+			<div class="popupCheckmark"></div>
+		</div>
+	`;
+	chrome.runtime.onMessage.addListener(	
+		function(request, sender, sendResponse) {	
+			if (request.command === "startTutorial") {	
+				$(tutorialPopupHtml).insertAfter($("body").children().first());
+				let tutorialPopup = $("#tutorialPopupContainer");
+				$(".popupCheckmark").css("background-image", "url("+popupCheckmarkUrl+")").click(function() {
+					$("#optionsButton").click();
+					tutorialPopup.remove()
+				});
+				$("#optionsButtonTutorial").click(function() {
+					$("#optionsButton").click();
+					tutorialPopup.remove();
+				})
+			}	
+		}	
+	);
+}
```

Comment:
<img width="1440" alt="Screen Shot 2020-08-03 at 2 32 03 PM" src="https://user-images.githubusercontent.com/47435096/89224764-a94a7680-d59e-11ea-9428-52a618244196.png">
<img width="1440" alt="Screen Shot 2020-08-03 at 2 31 57 PM" src="https://user-images.githubusercontent.com/47435096/89224768-ac456700-d59e-11ea-8a68-3fc80b0964a9.png">

Second image is what it looks like when you hover over the check mark. Clicking on either gear or the checkmark will get rid of tutorial box